src/main/java/Locators/ProductPageLocators.java

```diff
@@ -1,7 +1,87 @@
 package Locators;
 
-public class ProductPageLocators {
+import objects.Product;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import pages.BasePage;
+
+import java.util.ArrayList;
+
+public class ProductPageLocators extends BasePage {
+    private static String cart_item = "(//div[@class='inventory_item'])";
+
+    public static By label_product_img(int index) {
+        String img = "(//div[@class='inventory_item'])[1]//div[@class='inventory_item_img']";
```

Comment:
~"(//div[@class='inventory_item'])[1]//div[@class='inventory_item_img']";~
-> "(//div[@class='inventory_item'])[" + index + "]//div[@class='inventory_item_img']";

Reply:
Do the same for others